api/swagger.yaml

```diff
@@ -1496,10 +1496,21 @@ definitions:
         type: "string"
       progressDetail:
         $ref: "#/definitions/ProgressDetail"
+      aux:
+        $ref: "#/definitions/ImageID"
+
+  ImageID:
+     type: "object"
+     description: "Image ID or Digest"
+     properties:
+       ID:
+         type: "string"
```

Comment:
can you add example values to the properties you modified? That way we can (at some point) remove the response/request examples (which can easily get out of sync with the actual definition :smile:)

Reply:
ack.
Unfortunately other image related operation use `Id` (lowercase) while buildInfo use uppercase `ID`
https://github.com/moby/moby/blob/master/api/server/backend/build/backend.go#L59